src/inmanta/moduletool.py

```diff
@@ -528,7 +537,21 @@ def get_modules(self, module: Optional[str] = None) -> List[Module]:
         else:
             return self.get_project(load=True).sorted_modules()
 
-    def create(self, name: str) -> None:
+    def create(self, name: str, v1: bool, no_input: bool = False) -> None:
```

Comment:
We should add a comment to explain the the `no_input` argument is only relevant for V2 modules. Or we should change the name of the argument.

Reply:
Added a docstring, inlcuding this.